tidb-cloud/ticloud-import-cancel.md

```diff
@@ -1,48 +1,49 @@
 ---
-title: ticloud import cancel
-summary: The reference of `ticloud import cancel`.
+title: ticloud serverless import cancel
+summary: The reference of `ticloud serverless import cancel`.
 ---
 
-# ticloud import cancel
+# ticloud serverless import cancel
 
 Cancel a data import task:
 
 ```shell
-ticloud import cancel [flags]
+ticloud serverless import cancel [flags]
 ```
 
 ## Examples
 
 Cancel an import task in interactive mode:
 
 ```shell
-ticloud import cancel
+ticloud serverless import cancel
 ```
 
 Cancel an import task in non-interactive mode:
 
 ```shell
-ticloud import cancel --project-id <project-id> --cluster-id <cluster-id> --import-id <import-id>
+ticloud serverless import cancel --project-id <project-id> --cluster-id <cluster-id> --import-id <import-id>
 ```
 
 ## Flags
 
 In non-interactive mode, you need to manually enter the required flags. In interactive mode, you can just follow CLI prompts to fill them in.
 
-| Flag                    | Description                           | Required | Note                                               |
-|-------------------------|---------------------------------------|----------|-----------------------------------------------------|
-| -c, --cluster-id string | Cluster ID                            | Yes      | Only works in non-interactive mode.                   |
-| --force                 | Deletes a profile without confirmation | No       | Works in both non-interactive and interactive modes. |
-| -h, --help              | Help information for this command             | No       | Works in both non-interactive and interactive modes. |
-| --import-id string      | The ID of the import task                 | Yes      | Only works in non-interactive mode.                   |
-| -p, --project-id string | Project ID                            | Yes      | Only works in non-interactive mode.                   |
+| Flag                    | Description                            | Required | Note                                                 |
+|-------------------------|----------------------------------------|----------|------------------------------------------------------|
+| -c, --cluster-id string | Specifies the ID of the cluster.                             | Yes      | Only works in non-interactive mode.                  |
+| --force                 | Deletes a profile without confirmation. | No       | Works in both non-interactive and interactive modes. |
```

Comment:
"Deletes a profile without confirmation." looks incorrect here for this command

Reply:
fixed